plugins/module_utils/redfish_utils.py

```diff
@@ -89,12 +107,29 @@ def post_request(self, uri, pyld):
         req_headers = dict(POST_HEADERS)
         username, password, basic_auth = self._auth_params(req_headers)
         try:
-            resp = open_url(uri, data=json.dumps(pyld),
-                            headers=req_headers, method="POST",
-                            url_username=username, url_password=password,
-                            force_basic_auth=basic_auth, validate_certs=False,
+            #To support HPE operations
+            if "iLOlogin" in pyld.keys() or os.path.isfile(home + "/sessionfile.txt"):
```

Comment:
The `.keys()` is not needed. Also happens below.
```suggestion
            if "iLOlogin" in pyld or os.path.isfile(home + "/sessionfile.txt"):
```

Reply:
Changes have been made and is available in the latest commit.